sample-app/ios/Podfile.lock

```diff
@@ -1,6 +1,6 @@
 PODS:
   - boost-for-react-native (1.63.0)
-  - CocoaDebug (1.2.5)
+  - CocoaDebug (1.2.3)
```

Comment:
is there any reason to go back to `1.2.3` from `1.2.5`?

Reply:
My mistake